### PR TITLE
Need to add FAQ for sysctl command is missing

### PR DESCRIPTION
**Pull request by bvtejaswi**
### :nut_and_bolt: Description: What code changed, and why?
In case of failure while executing deploy / upgrade command we have to add steps in FAQ section to troubleshoot by themselves.

### :chains: Related Resources
https://chefio.atlassian.net/browse/MADROX-442

### :athletic_shoe: How to Build and Test the Change
https://deploy-preview-7746--chef-automate.netlify.app/

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: terraform/a2ha-terraform/modules/opensearch/templates/provision.sh.tpl [hunk 1]
```diff
@@ -61,7 +61,7 @@ if [ -e /hab/sup/default/specs/"$OS_PKG_NAME".spec ]; then
      #appliedConfigFile="config.$RANDOM.toml"
      #automate-backend-ctl applied --svc=automate-ha-opensearch | tail -n +2 > $appliedConfigFile
 
-     bash -c 'sysctl -w vm.max_map_count=262144'
+     bash -c 'hab pkg exec core/procps-ng sysctl -w vm.max_map_count=262144 "$LOGCMD"'
```
Comment (ArvinthC3000) [inline terraform/a2ha-terraform/modules/opensearch/templates/provision.sh.tpl:64]: Check if the core/procps-ng is installed at the time of executing this command. 
please add screenshot of file were the output is populated in the logs

Comment (bvtejaswi) [reply terraform/a2ha-terraform/modules/opensearch/templates/provision.sh.tpl]: Now the scope of this PR has changed

**Review by vivek-yadav — APPROVED** [review]

**Review by punitmundra — APPROVED** [review]

File: components/docs-chef-io/content/automate/ha_troubleshooting.md [hunk 2]
```diff
@@ -180,4 +180,23 @@ terraform taint module.bootstrap_automate.null_resource.automate_post[1]
 terraform taint module.bootstrap_automate.null_resource.automate_post[2]
 ```
 
+### Issue: Deployment or Upgrade Fails as UnknownError
 
+While trying to deploy or upgrade Chef Automate HA, if the command did not exit gracefully
+
+The possible error looks like as shown below:
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_troubleshooting.md:187]: While trying to deploy or upgrade Chef Automate HA, if the command did not exit gracefully, the possible error looks like as shown below:

(Merge these two lines)

File: components/docs-chef-io/content/automate/ha_troubleshooting.md [hunk 3]
```diff
@@ -180,4 +180,23 @@ terraform taint module.bootstrap_automate.null_resource.automate_post[1]
...
+
+```bash
+      ┬ Exiting as execution process completed
+UnknownError: Unable to determine error cause: Command did not exit gracefully
+```
+
+- Examine the logs from bastion (/hab/a2_deploy_workspace/logs/a2ha-run.log) or from frontend/backend nodes in svc-load.log (or) automate-ctl.log in /hab/var/automate-ha/
+- If you found that `sysctl: command not found`
+- Follow the below steps
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_troubleshooting.md:196]: If you find `sysctl: command not found`, follow the step below:

(merge these two pointers)

File: components/docs-chef-io/content/automate/ha_troubleshooting.md [hunk 4]
```diff
@@ -180,4 +180,23 @@ terraform taint module.bootstrap_automate.null_resource.automate_post[1]
...
+
+#### Solution
+
+- In logs check for which module it prompted an error.
+- SSH into the node and check if the `sysctl` utility is available by running `sysctl -a`.
+- If `Command 'sysctl' not found` , try installing the pkg.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_troubleshooting.md:202]: - If `Command 'sysctl' not found`, try installing the pkg. 


(remove the space before the comma)

**Review by dishanktiwari2501 — APPROVED** [review]
Done!

File: components/docs-chef-io/content/automate/ha_troubleshooting.md (hunk 2 above)
Comment (bvtejaswi) [reply components/docs-chef-io/content/automate/ha_troubleshooting.md:186]: Done

File: components/docs-chef-io/content/automate/ha_troubleshooting.md (hunk 3 above)
Comment (bvtejaswi) [reply components/docs-chef-io/content/automate/ha_troubleshooting.md]: Done

**Review by vivek-yadav — APPROVED** [review]

**Review by vivek-yadav — APPROVED** [review]

**Pull request merged**